src/content/docs/apm/agents/php-agent/installation/php-agent-installation-aws-linux-redhat-centos.mdx

```diff
@@ -18,7 +18,7 @@ Our PHP agent auto-instruments your code so you can start monitoring application
 
```

Comment:
Needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'